### PR TITLE
bump dependencies

### PR DESCRIPTION
**Pull request by adrianchiris**
- bump dependencies like in #577 
- fix lint issues by removing the use of grpc.WithBlock() dial option

### CODE REVIEW

**Comment by adrianchiris** [comment]
@SchSeba  @zeeke  PTAL mellanox ci failure is un-related

**Comment by adrianchiris** [comment]
@Eoghan1232 PTAL if you got time :)

**Review by Eoghan1232 — APPROVED** [review]
lgtm!

File: pkg/resources/server.go [hunk 1]
```diff
@@ -265,38 +264,14 @@ func (rs *resourceServer) Start() error {
 	}
 	pluginapi.RegisterDevicePluginServer(rs.grpcServer, rs)
 
+	// start serving from grpcServeer
```
[the remark this answers is not in the document]
Comment (adrianchiris) [reply pkg/resources/server.go:267]: thx, some of my keyboard switches are double typing :\

File: pkg/resources/server.go [hunk 2]
```diff
@@ -265,38 +264,14 @@ func (rs *resourceServer) Start() error {
...
 	go func() {
 		err := rs.grpcServer.Serve(lis)
 		if err != nil {
 			glog.Errorf("serving incoming requests failed: %s", err.Error())
 		}
 	}()
 
-	conn, err := grpc.NewClient(
```
[the remark this answers is not in the document]
Comment (adrianchiris) [reply pkg/resources/server.go:275]: we dont. thats an anti-pattern, see commit msg with relevant link.

**Review by SchSeba — APPROVED** [review]
LGTM

**Pull request merged**